pinaxcon/registrasion/management/commands/populate_inventory.py

```diff
@@ -99,7 +99,9 @@ def populate_inventory(self):
             inv.Category,
             ("name",),
             name="Sprint Ticket (Mon 7th - Tue 8th)",
-            description="A day of food, coffee and hacking",
+            description="Sprints are a day where you get to work with other members of the community. "
+                        "This is a great opportunity to spend some in-person time with members of the community. "
```

Comment:
in-person time is a bit ... ermhuh ... wonky.

This is a great opportunity to spend time with members of the community you might otherwise only "see" via the Internet.